web_src/js/features/repo-projects.ts

```diff
@@ -96,7 +97,7 @@ function initRepoProjectColumnEdit(writableProjectBoard: Element): void {
     });
   });
 
-  elForm.addEventListener('submit', async (e) => {
+  elForm.addEventListener('submit', async (e: SubmitEvent) => {
```

Comment:
I do not think it should keep adding such type.

The types could be inferred by

```
addEventListener<K extends keyof HTMLElementEventMap>(type: K, listener: (this: HTMLDivElement, ev: HTMLElementEventMap[K]) => any, options?: boolean | AddEventListenerOptions): void;

-> HTMLElementEventMap[K]

->  GlobalEventHandlersEventMap[K]

-> "submit": SubmitEvent;

```

Reply:
Yeah typescript does infer it correctly here, we can remove again.

Reply:
Oops, I see a bad case for this ....

It seems that TypeScript can't infer `async` listenser 🤣 , because "async"  doesn't match the defined types.

But anyway, since it passes the TS check, so I guess we do not need to write the types explicitly. So far so good.